astropy/modeling/parameters.py

```diff
@@ -590,8 +592,14 @@ def validator(self):
         should be raised, though this is not currently a requirement).
 
         """
+        # message = """The validator property of the Parameter class is deprecated.
```

Comment:
Not sure why this commented statement is left in. Perhaps a comment about using a validator decorator makes the model unpickleable is needed?